permissions/resource_permissions_test.go

```diff
@@ -455,37 +455,21 @@ func TestResourcePermissionsCustomizeDiff_ErrorOnCreate(t *testing.T) {
 		access_control {
 			permission_level = "WHATEVER"
 		}`,
-	}.ExpectError(t, "permission_level WHATEVER is not supported with cluster_id objects")
-}
-
-func TestResourcePermissionsCustomizeDiff_ErrorOnPermissionsDecreate(t *testing.T) {
```

Comment:
This behavior is changed in this PR: users are allowed to change their permissions as long as they maintain management-level permissions.